packages/xrpl/src/models/methods/ledgerEntry.ts

```diff
@@ -204,5 +211,9 @@ export interface LedgerEntryResponse<T = LedgerEntry> extends BaseResponse {
     /** The binary representation of the ledger object, as hexadecimal. */
     node_binary?: string
     validated?: boolean
+    /**
+     * (Optional) Indicates the ledger index at which the object was deleted.
+     */
+    deleted_ledger_index?: number
```

Comment:
Can you include unit and integration tests that demonstrate the working of this flag? Are these flags enabled on any Clio (or) rippled instances?

Reply:
This flag is for Clio only so I guess we don't need a test for it?

Reply:
@ckeshava there wouldn't be integration tests because this param is only available in clio, and currently the library doesn't support integration tests for clio

Reply:
I feel a unit test can be written to demonstrate the intended usage of `include_deleted` parameter.

Reply:
A unit test file does not exist for this file, but it can be created under `packages/xrpl/test/models`. This is not a hard requirement, so I'll approve this PR.